src/dbSta/src/dbNetwork.cc

```diff
@@ -867,6 +955,17 @@ void dbNetwork::readDbAfter(odb::dbDatabase* db)
       makeLibrary(lib);
     }
     readDbNetlistAfter();
+    // we make the library for the verilog hierarchical cells
+    // this is in the same fashion as the original dbInst code
+    // which uses the void* staGetCell to associate a cell with
+    // concrete cell. We do same for verilog hierarchical cells.
+    Library* verilog_library = makeLibrary("verilog", nullptr);
```

Comment:
Why do we need concrete cells for hierarchy modules?

Reply:
let's chat about this. We are simply following the infra-structure used by the STA which fishes out the cell link as a concrete cell from sta_cell pointer and likewise for ports. For exaple see mterm->staSetPort(cport) and master->staSetCell(reinterpret_cast<void*>(cell)); in dbNetwork.cc. We are following the same approach. I accept that approach has its faults, but it seems consistent.